bom/pom.xml

```diff
@@ -104,6 +104,7 @@
       <version.commons.logging>1.2</version.commons.logging>
       <version.commons.math>2.2</version.commons.math>
       <version.gnu.getopt>1.0.13</version.gnu.getopt>
+      <version.gson>2.8.2</version.gson>
```

Comment:
So I guess I'll need to update protostream to use the same gson version @gustavonalle

Reply:
Gson is shaded inside protostream @anistor, so I decided to use an explicit dependency so make it easier to upgrade

Reply:
right. forgot about the shading